ga-ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json

```diff
@@ -3449,5 +3449,50 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalAppRespondDebtorUploadLabel",
+    "UserRoles": [
+      "caseworker-civil-solicitor",
+      "caseworker-civil-systemupdate",
+      "judge-profile",
+      "legal-adviser",
+      "civil-administrator-standard",
+      "civil-administrator-basic",
+      "caseworker-ras-validation",
+      "ga-basic-access"
+    ],
+    "CRUD": "CRU"
```

Comment:
Giving CRU permission doesn't affect as it's a label type and displaycontext is readonly